src/main/java/com/bina/varsim/types/variant/Variant.java

```diff
@@ -611,7 +611,13 @@ public VariantType getType(final int ind) {
         we can be assured that correct variant type is
         returned.
          */
-        Alt alt = alts[ind - 1];
+        Alt alt = null;
+        try {
+            alt = alts[ind - 1];
+        } catch (IndexOutOfBoundsException e) {
+            log.error(this.toString());
+            throw new IndexOutOfBoundsException();
```

Comment:
just use `throw e;` to rethrow the same exception.

Reply:
Thanks. Fixed.